dissect/target/plugins/os/default/_os.py

```diff
@@ -50,6 +50,14 @@ def version(self) -> str | None:
     def users(self) -> Iterator[Record]:
         yield from ()
 
+    @internal
+    def misc_home_dirs(self) -> Iterator[tuple[str, tuple[str, str] | None]]:
```

Comment:
I'm not entirely sure if I like this being on the OS plugin. You'd need to add it to the base `OSPlugin` class too, if we are going that way though.

Maybe rename it to `misc_user_paths`, it feels a bit more generic (home directory is more a Unixy term).

Reply:
I thought about this over the weekend and I think having separate user plugins per OS is a bit too much, for now.

`misc_user_paths` doesn't feel  out of place to me in OSplugin (at least for now). 

Will add a comment to consider refactoring if more machinery is needed